app/connectors/IvConnector.scala

```diff
@@ -42,11 +41,21 @@ class IvConnector @Inject()(appConfig: FrontendAppConfig,
     val url = serviceUrl + environment
     infoLog(s"[IvConnector][checkJourneyLinkUplift] journeyId: $journeyId attempted")
     http.GET[JourneyLinkResponse](s"$url/mdtp/journey/journeyId/$journeyId")
+  }.recover{
+    case e: Exception =>
+      errorLog(s"[IvConnector][checkJourneyLinkUplift] journeyId: $journeyId Error: ${e.getMessage}")
+      JourneyLinkResponse("", "")
   }
```

Comment:
This is changing the behaviour of the method. Previously if the GET failed it would have thrown an exception. Now you are catching it and providing a response. How does this effect the behaviour of the things which call this method?